scripts/removeNonRemPlugin.js

```diff
@@ -1,27 +0,0 @@
-const postcss = require('postcss');
```

Comment:
Also remove this file 

https://github.com/AlaskaAirlines/auro-hyperlink/blob/main/scripts/prepForBuild.js